recipes/thriftpy2/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Also made an upstream PR to package the license: https://github.com/Thriftpy/thriftpy2/pull/54